clusterloader2/testing/prometheus/not-scrape-kube-proxy.yaml

```diff
@@ -1 +0,0 @@
-PROMETHEUS_SCRAPE_KUBE_PROXY: false
```

Comment:
Let's leave this file for now in case someone is using it, we can remove it later ;)